src/libs/vtools/dialogs/tools/piece/dialoginternalpath.cpp

```diff
@@ -614,16 +617,16 @@ void DialogInternalPath::EvalWidthBefore()
 //---------------------------------------------------------------------------------------------------------------------
 void DialogInternalPath::EvalWidthAfter()
 {
-    labelEditFormula = ui->labelEditAfter;
+    labelEditFormula = ui->afterWidthEdit_Label;
     const QString postfix = UnitsToStr(qApp->patternUnit(), true);
-    QString formula = ui->plainTextEditFormulaWidthAfter->toPlainText();
+    QString formula = ui->afterWidthForumla_PlainTextEdit->toPlainText();
```

Comment:
afterWidthForumla_PlainTextEdit

Reply:
You now what... Forumla... that was probably a Roman misspelling. :)